index.html

```diff
@@ -7,30 +7,44 @@
     <title>Porfolio - PDM</title>
     <link rel="stylesheet" href="index.css">
     <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
-</head>
-<!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-8PQYD54DJ1"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', 'G-8PQYD54DJ1');
-</script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-8PQYD54DJ1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        
+        gtag('config', 'G-8PQYD54DJ1');
+    </script>
+    </head>
 <body>
 <section class="page1">
-    <header>
-    <a href="#"><h1>Philip Modinos</h1></a>
-    <nav class="desktop">
-        <ul>
-            <li><a href="#portfolio">Portfolio</a></li>
-            <li><a href="#about">About</a></li>
-            <li><a href="#contact">Contact  <i class="fa fa-envelope-o" aria-hidden="true"></i></a></li>
-        </ul>
-    </nav>
-    <nav class="mobile">
-        <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
-    </nav>    
+    <header class="toggle">
+        
+            <a href="#"><h1>Philip Modinos</h1></a>
+            <!-- <nav class="toggle-mobile">
+                <i class="fa fa-times fa-2x" aria-hidden="true"></i>
+            </nav> -->
```

Comment:
- [ ] To make this file clean I think it would be good to remove commented code.